### PR TITLE
[kml] Correctly import bookmarks with track data

### PR DESCRIPTION
**Pull request by AndrewShkrob**
Closes: #3467

What do you think? Is it a valid solution?

TODO:
- [x] Cover this case with UTs

Here is how it looks inside a kml:
```xml
	<Placemark id="placemark20">
		<name><![CDATA[Кальянная Erenler Nargile ve Çay Bahçesi]]></name>
		<styleUrl>#icon20</styleUrl>
		<description><![CDATA[<table width="400"><tr><td></td></tr></table>]]></description>
		<MultiGeometry>
			<Point>
				<coordinates>28.968447783842,41.009030507129,0</coordinates>
			</Point>
			<LineString id="line20">
				<extrude>1</extrude>
				<tessellate>1</tessellate>
				<coordinates>28.968447783842,41.009030507129,0 28.965858,41.018449,0</coordinates>
			</LineString>
		</MultiGeometry>
	</Placemark>
```

### CODE REVIEW

File: kml/serdes.cpp [hunk 1]
```diff
@@ -745,7 +745,9 @@ void KmlParser::ParseAndAddPoints(MultiGeometry::LineT & line, std::string_view
 
 void KmlParser::ParseLineString(std::string const & s)
 {
-  m_geometryType = GEOMETRY_TYPE_LINE;
+  // If m_org is not empty, then it still a Bookmark but with track data
```
Comment (biodranik) [inline kml/serdes.cpp:748]:
```suggestion
  // If m_org is not empty, then it is still a Bookmark but with track data
```

**Review by biodranik — COMMENTED** [review]
Thanks!

1. Does it conform to the official KML specification?
2. Can you please update unit tests?

**Comment by AndrewShkrob** [comment]
>Does it conform to the official KML specification?

`MultiGeometry` can contain both lines and points, so yes, it's a valid kml.

>Can you please update unit tests?

Done

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 13

File: kml/kml_tests/serdes_tests.cpp [hunk 2]
```diff
@@ -874,3 +874,46 @@ UNIT_TEST(Kml_Ver_2_3)
   TEST_EQUAL(lines[0].size(), 7, ());
   TEST_EQUAL(lines[1].size(), 6, ());
 }
+
+UNIT_TEST(Kml_Placemark_contains_both_Bookmark_and_Track_data)
+{
+  char const * input = R"(<?xml version="1.0" encoding="UTF-8"?>
```
Comment (biodranik) [inline kml/kml_tests/serdes_tests.cpp:880]: nit: constexpr

File: kml/kml_tests/serdes_tests.cpp [hunk 3]
```diff
@@ -874,3 +874,46 @@ UNIT_TEST(Kml_Ver_2_3)
...
+    <LineString>
+      <coordinates>28.968447783842,41.009030507129,0 28.965858,41.018449,0</coordinates>
+    </LineString>
+    <Point>
+      <coordinates>28.968447783842,41.009030507129,0</coordinates>
+    </Point>
+  </MultiGeometry>
+</Placemark>
+</kml>
+  )";
```
Comment (biodranik) [inline kml/kml_tests/serdes_tests.cpp:903]:
```suggestion
</kml>)";
```

**Review by biodranik — APPROVED** [review]
LGTM, @vng WDYT?

**Pull request merged**